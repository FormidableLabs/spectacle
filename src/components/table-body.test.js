import React from "react";
import { mount } from "enzyme";
import { mountToJson } from "enzyme-to-json";
import TableBody from "./table-header";

describe("<TableBody />", () => {
  test("should render correctly", () => {
    const wrapper = mount(
      <TableBody>
        <tr><td>Table Content</td></tr>
      </TableBody>
    );
    expect(mountToJson(wrapper)).toMatchSnapshot();
  });
});
