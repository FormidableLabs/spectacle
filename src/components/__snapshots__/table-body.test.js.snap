exports[`<TableBody /> should render correctly 1`] = `
<TableHeader>
  <thead>
    <tr>
      <td>
        Table Content
      </td>
    </tr>
  </thead>
</TableHeader>
`;
